core/arxiv/submission/domain/event.py

```diff
@@ -261,6 +262,12 @@ def _primary_cannot_be_secondary(self, submission: Submission) -> None:
                                "The same category cannot be used as both the"
                                " primary and a secondary category.")
 
+    def _creator_must_be_endorsed(self, submission: Submission) -> None:
+        """The creator of this event must be endorsed for the category."""
+        if self.category not in self.creator.endorsements:
+            raise InvalidEvent(self,
+                               f"Creator is not endorsed for {self.category}")
```

Comment:
Does this error get raised to the log or to the user? If user, we might want to word this a bit differently.

Reply:
This could make it through to the user. Or we can override the message in the UI